dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/GitHubSourceCodeRepo.java

```diff
@@ -104,6 +104,7 @@
 public class GitHubSourceCodeRepo extends SourceCodeRepoInterface {
 
     public static final String OUT_OF_GIT_HUB_RATE_LIMIT = "Out of GitHub rate limit";
+    public static final int MAX_TOPIC_LENGTH = 150;
```

Comment:
In [Slack thread](https://app.slack.com/archives/C02PDJCUW69/p1638993155001500), @garyluu said the limit is 255 in the DB. Is there a reason this is smaller?

Reply:
I'll be changing it to 150 in my next PR. Seems in the mockups, everything is less than 150 characters. When we display it in the current UI (with our current font size and column size, it only fits about 100 characters)